package.json

```diff
@@ -129,6 +131,15 @@
     "test": "yarn test:app",
     "autorelease": "node scripts/autorelease.js",
     "prerelease": "node scripts/prerelease.js",
-    "release": "node scripts/release.js"
-  }
+    "release": "node scripts/release.js",
+    "size": "yarn build:app && size-limit"
+  },
+  "size-limit": [
```

Comment:
Per the PR description, could we put the limit in `src/packages/excalidraw/package.json`?  If not, I suspect #6037 would hit this limit due to the sizeable SVG font which MathJax ships as JS.  (That is one reason #6037 puts MathJax in `@excalidraw/extensions` instead of `@excalidraw/excalidraw`.)

Reply:
We can have two separate checks - one for the package `@excalidraw/excalidraw` and one for `excalidraw-app`. That way we can keep an eye on both :p

Reply:
Separate checks sound better to me.